### ISSUE TITLE
Port for GNURADIO 3.8

### ISSUE THREAD

**Issue by mohmmadd** [drmpeg/gr-dvbs2rx#8]
It is a very interesting OOT. Is there any plans to port this module to support GNU Radio 3.8?

**Comment by drmpeg** [comment]
https://github.com/drmpeg/gr-dvbs2rx38

**Comment by mohmmadd** [comment]
Excellent!, Thankyou.

**Comment by drmpeg** [comment]
I forgot to include the example flow graphs. I just pushed them.

**Comment by drmpeg** [comment]
If you're still interested in this OOT, I found a pretty severe bug in the port to GNU Radio 3.8. I neglected to update the compiler flags and the performance was horrible on x86.

Note that I've rearranged the repository. gr-dvbs2rx38 has been deleted and this repository contains versions for GNU Radio 3.7, 3.8 and 3.9 on the maint-3.7, maint-3.8 and master branches respectively.

To build for GNU Radio 3.8:

git clone https://github.com/drmpeg/gr-dvbs2rx.git
cd gr-dvbs2rx
git checkout maint-3.8
mkdir build
cd build
cmake ../
make
sudo make install
sudo ldconfig
